module/userProjects/helper.js

```diff
@@ -618,6 +618,34 @@ module.exports = class UserProjectsHelper {
 					}
 				}
 
+				let downloadableUrlsCall
+
+				if (projectDetails[0].story?.pdfInformation.length > 0) {
+					const pdfData = projectDetails[0].story
+
+					// Extract file paths from pdfInformation
+					const filePathUrls = pdfData.pdfInformation.map((path) => path.filePath)
+
+					// Get downloadable URLs
+					downloadableUrlsCall = await cloudServicesHelper.getDownloadableUrl(filePathUrls)
+
+					// Update the pdfInformation with corresponding URLs
+					if (downloadableUrlsCall?.result?.length > 0) {
+						pdfData.pdfInformation = pdfData.pdfInformation.map((info) => {
```

Comment:
@MallanagoudaB dynamically add one key to each pdf object - sharableUrl  and refer below logic to create the url
if (response && response.success && response.folderPath != '') {
					const downloadUrl = `${process.env.ELEVATE_PROJECT_SERVICE_URL}/${process.env.SERVICE_NAME}${CONSTANTS.endpoints.AUTO_DOWNLOAD}?file=${response.folderPath}`
					return resolve({
					
					function is userProjects/share